form.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Forms</title>
+</head>
+<body>
+<h1>Forms</h1>
+<form>
+    <label for="id-firstname">Firstname:</label>
+    <input type="text" name="firstname" id="id-firstname">
+    <hr>
+    <input type="text" name="lastname" placeholder="Lastname">
+    <hr>
+    <label for="filter1">First filter</label>
+    <input type="checkbox" name="filter" id="filter1" value="filter_1">
+    <label for="filter2">Second filter</label>
+    <input type="checkbox" name="filter" id="filter2" value="filter_1">
```

Comment:
You have the same values in two inputs. I think it's just a typo. Checkboxes should have different values.